packages/@aws-cdk/toolkit-lib/lib/api/io/private/codes.ts

```diff
@@ -109,6 +109,24 @@ export const CODES = {
     description: 'Confirm deploy security sensitive changes',
     level: 'info',
   }),
+
+  CDK_TOOLKIT_I5501: codeInfo({
+    code: 'CDK_TOOLKIT_I5501',
+    description: 'Stack Monitoring Start',
+    level: 'debug',
```

Comment:
Does `level` even make sense here? And does it make sense for it to be different from the "event" level?

Reply:
Does this imply the starting of a single stack, of the starting of an entire monitoring operation potentially containing multiple stacks?

If this relates to a single stack, don't we need context information to relate the start, stop and update events to each other?

Reply:
Don't we need `resourcesTotal` in the payload as well?

Reply:
> Does level even make sense here? And does it make sense for it to be different from the "event" level?

Currently every message has a level. It's defined as the "recommended printable output for a given verbosity". Acknowledging that different implementors might have a different opinion. But you are right, change it to `info`.

> If this relates to a single stack, don't we need context information to relate the start, stop and update events to each other?

Relates to a single stack. I just forgot documenting the interface. Nice catch!

Reply:
> Don't we need resourcesTotal in the payload as well?

Sure, why not.

Reply:
done